configuration/dot-env-changes.rst

```diff
@@ -52,7 +53,7 @@ changes can be made to any Symfony 3.4 or higher app:
    file. If you've customized this file, make sure to keep those changes (but use
    the rest of the changes).
 
-#. Update your `bin/console`_ (`bin/console diff`_) file to load the new ``config/bootstrap.php`` file.
```

Comment:
Removed the diff - because it wouldn't include the *latest* changes - and I think it's better just to point to the current file instead of trying to update the commit range.